src/routes/categories/components/CategoryTimeStamp.styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components'
+
+export const TimeStamp = styled.div`
+  font-family: ${props => props.theme.fonts.contentFontFamily};
+  font-size: ${props => props.theme.fonts.contentFontSize};
+  opacity: 0.5;
+  padding-top: 10px;
+`
```

Comment:
make it a `p`-tag, remove the padding-top and use textSecondaryColor instead of opacity

Reply:
I'll still leave the padding-top, imo the space in between should be bigger.